packages/core/strapi/bin/strapi.js

```diff
@@ -255,4 +257,157 @@ program
   .option('-s, --silent', `Run the generation silently, without any output`, false)
   .action(getLocalScript('ts/generate-types'));
 
+// `$ strapi export`
+const parseBool = (arg) => {
+  try {
+    return parseType({ type: 'boolean', value: arg });
+  } catch (e) {
+    return false;
+  }
+};
+// // Will be used for the options that accept a list
+// const listOption = (value) => {
+//   return value.split(',');
+// };
+
+program
+  .command('export')
+  .description('Export data from Strapi to file')
+  .option(
+    '--output <outputFilename>',
+    'Filename to output (without extension) [Default: "export_{yyyymmddHHMMSS}[.tar.gz][.gpg]"]'
+  )
+  .addOption(
+    new Option('--encrypt [boolean]', 'encrypt output file').default(true).argParser(parseBool)
+  )
+  .addOption(new Option('--no-encrypt', 'do not encrypt output file'))
+  .addOption(
+    new Option('--compress [boolean]', 'compress content').default(true).argParser(parseBool)
+  )
+  .addOption(new Option('--no-compress', 'do not compress content'))
```

Comment:
Why having both a "compress=boolean" & "no-compress" options?

Reply:
I was trying it out because Commander supports negations with the `--no-` prefix and left it in because it seemed intuitive for users who wanted to either use `--compress` or `--no-compress` instead of --compress=false. But I have no strong opinion on it, and removing them is easier than worrying about maintaining both.